src/ui/app.rs

```diff
@@ -43,6 +43,9 @@ impl App {
                 Menu::Send => ManagerSendState::new(revaultd).into(),
                 // Manager cannot delegate funds, the user is redirected to the home.
                 Menu::DelegateFunds => ManagerHomeState::new(revaultd).into(),
+                Menu::Settings => {
+                    SettingsState::new(self.revaultd.clone().map(|r| r.config.clone())).into()
```

Comment:
L36: ` let revaultd = self.revaultd.clone().unwrap();`
```suggestion
                    SettingsState::new(revaultd.config.clone()).into()
```